wordpress/dto/WordpressPostDTO.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2023. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+
+import { WordpressEnum } from "./WordpressTypesDTO";
+import { isString } from "../../types/String";
+import { isRegularObject } from "../../types/RegularObject";
+import { hasNoOtherKeysInDevelopment } from "../../types/OtherKeys";
+
+export interface WordpressPostDTO {
+    title:{rendered:string, protected:boolean};
```

Comment:
Would be better if these were readonly, but not sure if there is a good reason for this.

If there is a reason, it should be renamed as WritableWordpressPostDTO -- since editable types should be an exeception. Even in that case it would be best to provide readonly version also for cases where that is fine.